Nodejs/Product/TestAdapter/TestFrameworks/Jest/jest.js

```diff
@@ -0,0 +1,91 @@
+﻿// @ts-check
+"use strict";
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+
+const find_tests = function (testFileList, discoverResultFile, projectFolder) {
+    // Jest package is not required for this tests to run, but it's still good to check 
```

Comment:
Why is the Jest package not specifically required?

In the package.json discoverer we enumerate the installed packages to determine the test framework, so that won't work in this case?

Reply:
This is an invalid comment, the firsts commits didn't require the jest package specifically just the jest-editor-ssupport. That has changed now. I have removed the comment.